specification/schemas/StatusRelationship.json

```diff
@@ -1,5 +1,6 @@
 {
   "type": "object",
+  "readOnly": true,
```

Comment:
I don't think this should be read only. The payment application probably needs to be able to set the status by itself.